gradle/dependencies.gradle

```diff
@@ -160,7 +160,8 @@ ext {
       googleServices     : '4.3.3',
       mapboxSdkVersions  : '1.1.0',
       dokka              : '0.10.1',
-      mapbox             : '0.2.1'
+      mapboxSdkRegistry  : '0.4.0',
+      mapboxAccessToken  : '0.2.1'
```

Comment:
Confirmed with @pengdev that the Gradle plugins are using separate versions now.